Gemfile

```diff
@@ -24,18 +24,19 @@ gem 'newrelic_rpm'
 gem 'nokogiri'
 gem 'output-templates', '~> 4.4.3', github: 'guidance-guarantee-programme/output-templates'
 gem 'phoner'
-gem 'princely'
 gem 'postcodes_io'
+# Switch back to the original after this is merged:
+# https://github.com/mbleigh/princely/pull/52
+gem 'princely', github: 'AndrewVos/princely', branch: 'remove-alias-method-chain'
```

Comment:
As a general rule - if you do the work on pension wise's clock you should maintain those changes on our fork, not your personal one.